layouts/partials/nav.ejs

```diff
@@ -9,23 +9,29 @@
     <div class="attribution">
       <p>
         <% if (locals.byline) { %>
-          By <span class="author"><%= locals.byline %></span>.
+          By <span class="author"><%= byline %></span>.
         <% } else if (locals.createdBy && !locals.byline) { %>
-          Created by <span class="author"><%= createdBy %></span> <%= createdAt %>.
+          Created by <span class="author"><%= createdBy %></span><span class="date" data-date="<%= locals.createdAt %>"></span>.
         <% } %>
         <% if (locals.lastUpdatedBy) { %>
-          <%# just encode the last updated date as data property to make relative time work for cache %>
-      Last edited by <span class="author"><%= lastUpdatedBy %></span> <span class="last-updated" data-last-updated="<%= modifiedAt %>"></span>.</p>
-          <!-- populate last updated time -->
-          <script>
-          (function () {
-            var $lastUpdated = $('.last-updated')
-            var utcUpdated = $lastUpdated.data('last-updated')
-            var updatedMoment = moment(utcUpdated)
-            $lastUpdated.html(updatedMoment.fromNow())
-          })()
-          </script>
+          </p><p>
+            Last edited by <span class="author"><%= lastUpdatedBy %></span><span class="date" data-date="<%= locals.modifiedAt %>"></span>.</p>
+        <% } %>
+        <% if (locals.byline) { %>
+          <p>Created <span class="date" data-date="<%= locals.createdAt %>"></span>.</p>
         <% } %>
+        <script>
+          (function() {
```

Comment:
I would put the comment back explaining why these are embedded as data properties just to make it clear why it's being done this way.